feg/gateway/services/session_proxy/credit_control/gy/config.go

```diff
@@ -207,20 +210,23 @@ func GetGyGlobalConfig() *GyGlobalConfig {
 	configsPtr := &mconfig.SessionProxyConfig{}
 	err := managed_configs.GetServiceConfigs(credit_control.SessionProxyServiceName, configsPtr)
 	siStr := diameter.GetValueOrEnv(OCSServiceIdentifierFlag, OCSServiceIdentifierEnv, "")
+	avp437 := diameter.GetValueOrEnv(OCSServiceIdentifierFlag, DisableRequestedGrantedUnitsAVPEnv, "")
```

Comment:
I think, it may be cleaner to make it bool, you can do something like this:

    avp437 := *avp437Flag || util.IsTruthyEnv(DisableRequestedGrantedUnitsAVPEnv)

Reply:
nice